examples/known-problems/fixed/FunctionProblemFixed1.pvl

```diff
@@ -0,0 +1,13 @@
+//:: cases FunctionProblemFixed1
+//:: tools silicon
+//:: verdict Error
+
```

Comment:
In examples/known-problems/FunctionProblem.pvl there was a comment explaining why this fails. It may be since to include it. As "FunctionProblemFixed" is not as descriptive.